Blockzilla/BrowserViewController.swift

```diff
@@ -7,7 +7,37 @@ import UIKit
 import SnapKit
 import Telemetry
 
-class BrowserViewController: UIViewController {
+class BrowserViewController: UIViewController, UIDragInteractionDelegate {
+
+    func dragInteraction(_ interaction: UIDragInteraction, sessionDidMove session: UIDragSession) {
```

Comment:
Could you move these functions into an extension that conforms to the protocol? (example: https://github.com/sblatz/focus-ios/blob/d40bfdd5e42eddd4280ac7fb5bc9b29965210963/Blockzilla/BrowserViewController.swift#L702)